pdm/doc/build/action.yml

```diff
@@ -16,7 +16,8 @@ inputs:
     deprecationMessage: use JFrog instead
   openapi:
     description: Whether or not to build OpenAPI specs
```

Comment:
Add an explanation about non `true` or `false` value, that it needs to be a job artifact name